modules/apps/object/object-js-components-web/src/main/resources/META-INF/resources/utils/api.ts

```diff
@@ -94,20 +94,20 @@ interface ObjectRelationship {
 	type: ObjectRelationshipType;
 }
 
-interface PickListItem {
+interface ListTypeEntry {
```

Comment:
@igor-franca  I think we'll need to sort this interface and the one below.

Reply:
It has already been done in the sort commit